api/capi/src/nnstreamer-capi-single.c

```diff
@@ -153,7 +153,7 @@ ml_single_open (ml_single_h * single, const char *model,
   status = ml_check_nnfw_availability (nnfw, hw, &available);
   if (status != ML_ERROR_NONE || !available) {
     ml_loge ("The given nnfw is not available.");
-    return status;
+    return ML_ERROR_NOT_SUPPORTED;
```

Comment:
No.

```return ML_ERROR_NOT_SUPPORTED``` only when available == FALSE.
Otherwise, if status != ML_ERROR_NONE, return status.

Reply:
I updated this line, please re-sync the changes.